Neos.Media.Browser/Classes/Service/FileTypeIconService.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Neos\Media\Browser\Service;
+
+/*
+* This file is part of the Neos.Media.Browser package.
+*
+* (c) Contributors of the Neos Project - www.neos.io
+*
+* This package is Open Source Software. For the full copyright and license
+* information, please view the LICENSE file which was distributed with this
+* source code.
+*/
+
+/**
+ * Service that retrieves an icon for the file type of a given filename
+ */
+class FileTypeIconService
```

Comment:
Doesn't this exist in the Media package?

Reply:
Yes, in a slightly different way (it needs Asset objects instead of a filename). I'll move this to the Media package and just the code accordingly.